src/cmd/apply.ts

```diff
@@ -120,7 +126,8 @@ const applyAll = async () => {
         { streams: { stdout: d.stream.log, stderr: d.stream.error } },
       )
       await cloneOtomiChartsInGitea()
-      await retryCheckingForPipelinerun()
+      await retryCheckingForPipelineRun()
+      await retryIsOAuth2ProxyRunning()
```

Comment:
Just an idea, would be better to check if well_known url from keycloak is responsive?
```
https://keycloak.<domainSuffix>/realms/otomi/.well-known/openid-configuration
```

It seems that checking if Oauth2Proxy Pod is running may not be enough

Reply:
we agreed to check oauth2 endpoint